libsolidity/interface/Natspec.cpp

```diff
@@ -80,7 +81,21 @@ Json::Value Natspec::userDocumentation(ContractDefinition const& _contractDef)
 				}
 			}
 		}
+
+	for (auto const& event: _contractDef.events())
+	{
+		string value = extractDoc(event->annotation().docTags, "notice");
+		if (!value.empty())
+		{
+			Json::Value user;
+			user["notice"] = Json::Value(value);
+			events[event->functionType(true)->externalSignature()] = user;
+		}
```

Comment:
Doesn't
```suggestion
    events[event->functionType(true)->externalSignature()]["notice"] = value;
```
work as well?